public/js/logix.js

```diff
@@ -1102,10 +1102,8 @@ CircuitElement.prototype.fixDirection = function() {
 // NOT OVERIDABLE
 CircuitElement.prototype.isHover = function() {
 
-    var mouseX = simulationArea.mouseX;
-    var mouseY = simulationArea.mouseY;
-
-    if (Math.abs(mouseX - this.x > Math.max(this.leftDimensionX, this.rightDimensionX)) || Math.abs(mouseY - this.y > Math.max(this.upDimensionY, this.downDimensionY))) return false;
```

Comment:
I wonder what this snippet of code was doing. @fpintos did it do anything useful?

Reply:
I believe the intention was to shortcut the calculation if the mouse was not inside a bounding rectangle around the element; but the expression was in fact incorrect, since the closing parenthesis for Math.abs was in the wrong place.
In the end, the final expression does the same with the correct values, so this was not needed at all.